tests/test_voc_format.py

```diff
@@ -1127,3 +1127,32 @@ def categories(self):
         with TestDir() as test_dir:
             self._test_save_and_load(TestExtractor(),
                 partial(VocConverter.convert, label_map='voc'), test_dir)
+
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
+    def test_can_save_dataset_without_bg_label(self):
+        expected = Dataset.from_iterable([
+            DatasetItem(1, subset='test', image=np.ones((2, 3, 3)),
+                annotations=[
+                    Bbox(0, 1, 0, 0, label=1, group=1, id=1, attributes={
+                        'truncated': False,
+                        'difficult': False,
+                        'occluded': False,
+                    })
+                ]),
+        ], categories={
+            AnnotationType.label: LabelCategories.from_iterable(
+                ['a', 'b']),
+            AnnotationType.mask: MaskCategories(
+                colormap=VOC.generate_colormap(2)),
```

Comment:
Probably, you need to remove the background color to test it. Consider using `MaskCategories.generate(include_background=False)`

Reply:
If the background label will always be added for masks, do we need to change something for the mask categories?

Reply:
I don't understand the question. Now, the background label is `a` in the `expected` dataset.